sky/serve/replica_managers.py

```diff
@@ -490,15 +491,12 @@ def probe(
             readiness_path = (f'http://{self.url}{readiness_path}')
             if post_data is not None:
                 msg += 'POST'
-                response = requests.post(
-                    readiness_path,
-                    json=post_data,
-                    timeout=serve_constants.READINESS_PROBE_TIMEOUT_SECONDS)
+                response = requests.post(readiness_path,
+                                         json=post_data,
+                                         timeout=timeout)
```

Comment:
Should we use the timeout for the post request directly or should we instead handle the timeout ourselves for when to set the replica to be `NOT_READY`? Reason: changing the timeout for the post request will cause the probing blocked by the a single request.

Reply:
Could you elaborate on `handle the timeout ourselves for when to set the replica to be NOT_READY`..? Do you refer to letting the user configure the consecutive failure timeout?

Reply:
Though it is from a user's request